spec/unit/resource_controller/forms_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+describe ActiveAdmin::ResourceController::Forms do
+  let(:controller_class) do
+    Class.new do
+      def self.name
+        "Test Controller using Forms"
+      end
+
+      include ActiveAdmin::ResourceController::Forms
+
+      public :apply_form
+    end
+  end
+
+  let(:controller) { controller_class.new }
+  let(:active_admin_config) { double(form_class: form_class) }
+  before do
+    allow(controller).to receive(:active_admin_config)
+                         .and_return(active_admin_config)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 21) spaces for indenting an expression spanning multiple lines.